extensions/ql-vscode/src/variant-analysis/variant-analysis-monitor.ts

```diff
@@ -119,8 +123,13 @@ export class VariantAnalysisMonitor extends DisposableObject {
         continue;
       }
 
+      // Get the current status of the variant analysis as known by the rest
+      // of the app, because it may have changed by the user, but the API
```

Comment:
Very optional suggestion which I think makes the comment slightly more correct, but I leave the decision up to you if you want to accept this or something else or do nothing:
```suggestion
      // of the app, because it may have been changed by the user and this code
```